Translations to estonian

Ottr-arch opened the pull request:
Hi Luke, great project! Good instructions and easy to set up.
I added the estonian translations. 
I found the AQI (EUROPE_CAQI) part confusing as "very low" is supposed to mean the quality is good :D. So I translated the "Air Quality Index" as just "Air Quality" in the oppopsite way from "very bad" to "very good" in estonian.

![esp32-weather](https://github.com/lmarzen/esp32-weather-epd/assets/80347747/19da4ac3-c8b4-48e6-a527-8738a38e4b3a)

lmarzen commented:
> I found the AQI (EUROPE_CAQI) part confusing as "very low" is supposed to mean the quality is good :D. So I translated the "Air Quality Index" as just "Air Quality" in the oppopsite way from "very bad" to "very good" in estonian.

In regards to this, I think the confusion come from the fact that air quality scales try to answer one of two questions...

1. How polluted is the air? (ex: European CAQI scale)
2. How good is the air quality? (ex: United States AQI scale)

It is not always clear what question an AQI scale is trying to answer unless you refer to the governing documents.

Here is another issue on this project where a similar question is asked, https://github.com/lmarzen/esp32-weather-epd/issues/31. I have quoted part of my reply here for reference.

> I used the Qualitative Name/Descriptor/Category outlined in the official standard for each AQI. Some AQI's attempt to fulfil different purposes, some present amount of air pollution like CAQI, and others like the US AQI present air quality closer to the sense that you are suggesting.
> 
> For the EU's CAQI...
> 
> from [http://airqualitynow.eu/about_indices_definition.php](https://web.archive.org/web/20220324185850/http://airqualitynow.eu/about_indices_definition.php)
> and
> from [http://www.airqualitynow.eu/download/CITEAIR-Comparing_Urban_Air_Quality_across_Borders.pdf](https://web.archive.org/web/20220303184040/http://www.airqualitynow.eu/download/CITEAIR-Comparing_Urban_Air_Quality_across_Borders.pdf)
> 
>     These indices have five levels using a scale from ‘0‘ (very low) to ‘> 100’ (very high). It presents a relative measure of the amount of air pollution.
> 
> For the US's AQI...
> 
> from https://www.airnow.gov/sites/default/files/2020-05/aqi-technical-assistance-document-sept2018.pdf
> 
>     The purpose of the AQI is to inform people about their air quality so they can take steps to protect their health.

Ottr-arch commented:
Hi Luke. Thank you for the clarification. Yes, I found the same answers with my research because I also thought that it might be a bug, then I realized that it essentially represents the pollution level (why did they not name it like that? :D ) from low to high. 
Which is why I renamed the field and removed the "Index" part to just say "Air quality" represented from "very bad" to "very good".
If you like, I can provide the correct translations and representation according to the correct CAQI format in Estonian and keep these current texts (which I find easier to understand) on my local build.

lmarzen commented:
You're right, this is definitely confusing as is. Maybe the label "Air Quality" needs to change depending on the scale selected. For EU CAQI the label should be "Pollution" or "Pollution Level". I'm going to add some code so the label used will depend on what AQI is selected. Then you can correct the translation.

lmarzen commented on platformio/src/locales/locale_et_EE.inc:
```diff
@@ -0,0 +1,438 @@
+const char *TXT_HUMIDITY           = "\xD5huniiskus";
+const char *TXT_UV_INDEX           = "UV Indeks";
+const char *TXT_PRESSURE           = "R\xF5hk";
+const char *TXT_AIR_QUALITY        = "\xD5hu Kvaliteet";
+const char *TXT_AIR_POLLUTION      = "\xD5husaaste";
```
Please make sure the air pollution translation makes sense here (I used google translate haha).

Ottr-arch replied:
It does, all good 👍

lmarzen commented:
Based on your suggestion, I have reworked the code so that depending on the AQI scale being used, it will either say "Air Quality" or "Air Pollution.". I updated your pull request with the new fields so that it is compatible with the new changes to upstream.

You should be good to update the translations back to the true meanings and it should be more clear now.

Ottr-arch commented:
It was a good workaround 👍 
Please look if all is good now, I'm new to git (neither am I a developer :D ) as this was my first pull request ever.

lmarzen commented:
Looks great! Give yourself a pat on the back for navigating it all like a pro, pretty impressive for a non-developer I would never have known :)

Thank you for your contribution.
